streams/src/main/java/org/apache/kafka/streams/state/internals/TimeOrderedKeySchema.java

```diff
@@ -43,12 +45,12 @@
 
     @Override
     public Bytes upperRange(final Bytes key, final long to) {
-        throw new UnsupportedOperationException();
+        return null;
```

Comment:
Should we document in the `KeySchema` interface that upperRange and lowerRange may return null when it is not supported by the impl?

Reply:
Works for me. @guozhangwang WDYT?